src/pwa-buildpack/project-setup/install-dependencies/index.md

```diff
@@ -0,0 +1,100 @@
+
+[previous topic]: {{ site.baseurl }}{% link pwa-buildpack/project-setup/create-theme-files/index.md %}
+
+## Initialize the package configuration file
+
+1. The following command runs a tool that walks you through the process of creating your project's `package.json` file:
+   
+   ``` bash
+   npm init
+   ```
+   
+1. For this tutorial, leave the input blank and hit `Return` or `Enter` on all the prompts to accept the default values.
+
+## Install project dependencies
+
+1. Run the following command to install the `dotenv` module:
+
+   ``` bash
+   npm install --save-dev dotenv
+   ```
+
+   This module loads environment values from a `.env` file and allows you to separate environment-specific values from source code.
+
+1. Run the following command to install [peregrine] and [React] as a production dependency:
+
+   ``` bash
+   npm install --save @magento/peregrine \
+   react react-dom react-redux react-router-dom redux
+   ```
+
+   These dependencies allow you to use the UI components and services provided by the peregrine project to develop your theme.
+
+1. Run the following command to install [pwa-buildpack] and its standard peer dependincies:
+
+   ``` bash
+   npm install --save-dev @magento/pwa-buildpack \
+   babel-core babel-loader babel-helper-module-imports \
+   babel-plugin-syntax-jsx babel-plugin-transform-class-properties \
+   babel-plugin-transform-object-rest-spread babel-plugin-transform-react-jsx \
+   webpack webpack-cli webpack-dev-server
+   ```
+
+   These dependencies allow you to create a [Webpack]-based build environment for your project.
+
+   It also installs [Babel], a JavaScript transpiler that allows you to write code using the latest and custom JavaScript language features without having to worry about browser compatibility.
+
+## Review `package.json` content
+
+Now that you have your project dependencies installed, your `package.json` file should look like the following:
+
+``` json
+{
+  "name": "orange-theme",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@magento/peregrine": "^0.1.5-alpha",
+    "react": "^16.3.1",
+    "react-dom": "^16.3.1",
+    "react-redux": "^5.0.7",
+    "react-router-dom": "^4.2.2",
+    "redux": "^3.7.2"
+  },
+  "devDependencies": {
+    "@magento/pwa-buildpack": "^0.2.0",
+    "babel-core": "^6.26.0",
+    "babel-helper-module-imports": "^7.0.0-beta.3",
+    "babel-loader": "^7.1.4",
+    "babel-plugin-syntax-jsx": "^6.18.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "dotenv": "^5.0.1",
+    "webpack": "^4.5.0",
+    "webpack-cli": "^2.0.14",
+    "webpack-dev-server": "^3.1.3"
+  }
+}
+```
+
```

Comment:
Here's where we should add the part about installing it into Magento.

>## Link theme files with Magento 2 store with Composer
>
>1. Change directories back to the Magento 2 application root.
>
>2. Run the following command to add your theme directory to
>   `composer.json` as a module repository.
>
>   ```sh
>   composer config repositories.pwadev \
>   '{"type":"path","url":"path/to/orange-theme","options":{"symlinks":true}}'
>   ```
>
>3. Run the following command to add your theme to `composer.json`
>   as a local linked dependency.
>
>   ```sh
>   composer require --dev orangecompany/orange-theme
>   ```
>
>4. Your `orange-theme` directory should now be symlinked to
>   `vendor/orangecompany/orange-theme` in your Magento application root 
>   directory. Confirm this by running the following command in your
>   Magento application root:
>
>   ```sh
>   ls -og vendor/orangecompany | grep "^l"
>   ```
>
>   You should see a listing that indicates the link to your real `orange-theme`
>   directory:
>
>   ```sh
>   lrwxr-xr-x   1    37 Apr 11 21:54 orange-theme -> path/to/orange-theme
>   ```
>
>   Composer has installed your theme to your Magento store with a symlink. You
>   can now edit your theme in its original location and your changes will
>   appear immediately in your store.

Reply:
We also have to comment that this will only work (presently) with the not-production-ready Magento `2.3-develop` branch. We have an issue pending to detect the absence of GraphQL and simulate for developer convenience.